lib/request.js

```diff
@@ -292,8 +291,7 @@ module.exports = {
    */
 
   get idempotent() {
-    const methods = ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'];
-    return !!~methods.indexOf(this.method);
+    return ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'].includes(this.method);
```

Comment:
What about `return this.method == 'GET' || this.method == 'HEAD' || this.method == 'PUT' || this.method == 'DELETE' || this.method == 'OPTIONS' || this.method == 'TRACE';`